model_specific/SM/standard_model.cpp

```diff
@@ -4900,6 +4858,24 @@ std::ostream& operator<<(std::ostream& ostr, const Standard_model& model)
    return ostr;
 }
 
+double Standard_model::calculate_alpha_s_SM5_at(
+   softsusy::QedQcd qedqcd_tmp, double scale) const
+{
+   qedqcd_tmp.run_to(scale); // running in SM(5)
+   return qedqcd_tmp.displayAlpha(softsusy::ALPHAS);
+}
+
+void Standard_model::calculate_MVWp_pole_fit(const double mh_pole) {
```

Comment:
Please replace `const double mh_pole` by `double mh_pole` in the function argument list.

Reply:
Ok. Side note: I actually like this syntax. Even though simple types are passed via copy, making them const does not allow you to change the value of the copy. So you know that if you use this variable, it's exactly what was passed to the function. Which is nice for very long functions that may try to do something with the passed argument before you come to using it.

Reply:
Hmm, that's a valid point. Ok, then we could leave it as it is, if you prefer. :)